### DISCUSSION TITLE
Got 'Coordinates are out-of-bounds' error  when updating to 0.9.5

### DISCUSSION THREAD

**Discussion by sjsillan** [OpenOrienteering/mapper#1969] [Questions & Answers] [answered]
I updated my Mapper to the version 0.9.5 . When I tried to open a map created with a previous Mapper versions, I got an error message 'Coordinates are out-of-bounds' and was unable to edit the file.

Does anybody have same kind of issues or knows a handy workaround for solving my problem?

**Comment by dl3sdo** [comment]
Can you provide more information about the issue:
- which was the previous version of Mapper?
- which OS are you using?
- what is the CRS of your map?

I opened georef. maps created with older versions of Mapper with 0.95 and did not encounter an error message like yours.

**Reply by sjsillan** [reply]
I'll share the map file which caused the issue.
[sample_map.zip](https://github.com/OpenOrienteering/mapper/files/6669087/sample_map.zip)

I am using Ubuntu 20.04. The previous Mapper version was 0.9.2 and CRS WGS84.

**Reply by dl3sdo** [reply]
Thank you for providing the map.
I tested it with Mapper 0.92, 0.93, 0.94 and 0.95 on Windows and I got the same error message for every version, including 0.92.

**Comment by dg0yt** [comment, marked as answer]
Thanks for reporting the issue, and sorry for the inconvenience. The offending coordinates are in an inactive template configuration. The bug is probably related to the change which introduced asynchronous template loading.

There are also some other strange things happening with regard to georeferencing. The data in the file declares a declination of 189° and a grivation of 192°. This is off by 180°, and I haven't seen this before. After  working around the coordinate issue and loading the map, the geographic coordinates end up somewhere imnthe Sahara. Oh...

Shall I attach the loadable map here?

**Reply by sjsillan** [reply]
Many thanks for helping me with the issue and your great work with the OOM project!

Sure, you can attach the loadable map here.
